homeassistant/components/enphase_envoy/quality_scale.yaml

```diff
@@ -0,0 +1,96 @@
+rules:
+  # Bronze
+  action-setup:
+    status: done
+    comment: only actions implemented are platform native ones.
+  appropriate-polling:
+    status: done
+    comment: fixed 1 minute cycle based on Enphase Envoy device characteristics
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
+  docs-actions:
+    status: done
+    comment: https://www.home-assistant.io/integrations/enphase_envoy/#actions
+  docs-high-level-description:
+    status: done
+    comment: https://www.home-assistant.io/integrations/enphase_envoy
+  docs-installation-instructions:
+    status: done
+    comment: https://www.home-assistant.io/integrations/enphase_envoy#prerequisites
+  docs-removal-instructions:
+    status: done
+    comment: https://www.home-assistant.io/integrations/enphase_envoy#removing-the-integration
+  entity-event-setup:
+    status: done
+    comment: no events used.
+  entity-unique-id: done
+  has-entity-name: done
+  runtime-data: done
+  test-before-configure: done
+  test-before-setup: done
+  unique-config-entry: done
+
+  # Silver
+  action-exceptions:
+    status: todo
+    comment: |
+      needs to raise appropriate error when exception occurs.
+      Pending https://github.com/pyenphase/pyenphase/pull/194
+  config-entry-unloading: done
+  docs-configuration-parameters:
+    status: done
+    comment: https://www.home-assistant.io/integrations/enphase_envoy#configuration
+  docs-installation-parameters:
+    status: done
+    comment: https://www.home-assistant.io/integrations/enphase_envoy#required-manual-input
+  entity-unavailable: done
+  integration-owner: done
+  log-when-unavailable: done
+  parallel-updates:
+    status: todo
+    comment: pending https://github.com/home-assistant/core/pull/132373
+  reauthentication-flow: done
+  test-coverage: done
+
+  # Gold
+  devices: done
+  diagnostics: done
+  discovery-update-info: done
+  discovery: done
+  docs-data-update:
+    status: done
+    comment: https://www.home-assistant.io/integrations/enphase_envoy#data-updates
+  docs-examples:
+    status: todo
+    comment: add blue-print examples, if any
+  docs-known-limitations: todo
+  docs-supported-devices:
+    status: done
+    comment: https://www.home-assistant.io/integrations/enphase_envoy#supported-devices
+  docs-supported-functions: todo
+  docs-troubleshooting:
+    status: done
+    comment: https://www.home-assistant.io/integrations/enphase_envoy#troubleshooting
+  docs-use-cases: todo
+  dynamic-devices: todo
+  entity-category: todo
+  entity-device-class: done
+  entity-disabled-by-default: done
+  entity-translations: done
+  exception-translations:
+    status: todo
+    comment: pending https://github.com/home-assistant/core/pull/132483
+  icon-translations: todo
+  reconfiguration-flow: done
+  repair-issues:
+    status: done
+    comment: authentication failures show as repair
```

Comment:
Those are not the ones we mean here, we mean the general issues created in the issue registry or repair issues via repairs.py and that's not the case here

Reply:
Exempt